centos/8/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM centos as builder
```

Comment:
Could you stick with `alpine`? It is sooo much smaller, making development and build experience much better.

Reply:
I agree.
I had some trouble consuming https://cloud.centos.org/centos/8/x86_64/images/CHECKSUM in alpine. Do you have an idea how to check it in alpine?

Reply:
`apk add wget` probably solves it. The `wget` included in alpine by default is not really `wget`.

Reply:
```
[dominik@p1gen3 containerdisks]$ podman run --rm -t -i alpine
/ # apk add wget
(1/3) Installing libunistring (0.9.10-r1)
(2/3) Installing libidn2 (2.3.2-r0)
(3/3) Installing wget (1.21.2-r2)
Executing busybox-1.34.1-r3.trigger
OK: 8 MiB in 17 packages
/ # cd /tmp/
/tmp # wget https://cloud.centos.org/centos/8-stream/x86_64/images/CentOS-Stream
-GenericCloud-8-20200113.0.x86_64.qcow2
--2021-11-25 13:43:55--  https://cloud.centos.org/centos/8-stream/x86_64/images/CentOS-Stream-GenericCloud-8-20200113.0.x86_64.qcow2
Resolving cloud.centos.org (cloud.centos.org)... 3.8.21.190, 2a05:d01c:c6a:cc01:11ae:4a43:febc:bd33
Connecting to cloud.centos.org (cloud.centos.org)|3.8.21.190|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 739962368 (706M) [application/octet-stream]
Saving to: 'CentOS-Stream-GenericCloud-8-20200113.0.x86_64.qcow2'

CentOS-Stream-Gener 100%[===================>] 705.68M  29.2MB/s    in 26s     

2021-11-25 13:44:21 (27.4 MB/s) - 'CentOS-Stream-GenericCloud-8-20200113.0.x86_64.qcow2' saved [739962368/739962368]

/tmp # wget https://cloud.centos.org/centos/8-stream/x86_64/images/CHECKSUM
--2021-11-25 13:44:30--  https://cloud.centos.org/centos/8-stream/x86_64/images/CHECKSUM
Resolving cloud.centos.org (cloud.centos.org)... 3.8.21.190, 2a05:d01c:c6a:cc01:11ae:4a43:febc:bd33
Connecting to cloud.centos.org (cloud.centos.org)|3.8.21.190|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 4008 (3.9K)
Saving to: 'CHECKSUM'

CHECKSUM            100%[===================>]   3.91K  --.-KB/s    in 0s      

2021-11-25 13:44:30 (61.4 MB/s) - 'CHECKSUM' saved [4008/4008]

/tmp # grep CentOS-Stream-GenericCloud-8-20200113.0.x86_64.qcow2 CHECKSUM
# CentOS-Stream-GenericCloud-8-20200113.0.x86_64.qcow2: 722624K 
SHA256 (CentOS-Stream-GenericCloud-8-20200113.0.x86_64.qcow2) = b80ca9ccad8715aa1aeee582906096d3190b91cebe05fcb3ac7cb197fd68a7fe
/tmp # grep CentOS-Stream-GenericCloud-8-20200113.0.x86_64.qcow2 CHECKSUM > c2
/tmp # sha256sum -c c2
sha256sum: WARNING: 2 of 2 computed checksums did NOT match

```

Reply:
I think that you are feeding this line into the sha256sum:

```
SHA256 (CentOS-Stream-GenericCloud-8-20200113.0.x86_64.qcow2) = b80ca9ccad8715aa1aeee582906096d3190b91cebe05fcb3ac7cb197fd68a7fe
```

Is this something which `s256sum` can understand? As far as I know this is the format it needs:

```
b80ca9ccad8715aa1aeee582906096d3190b91cebe05fcb3ac7cb197fd68a7fe  CentOS-Stream-GenericCloud-8-20200113.0.x86_64.qcow2
```

Note that there are two spaces between the shasum and the filename.

Reply:
I just learned that there is a BSD style checksum and a GNU style one. Looks like CentOS is using the BSD style one, while alpine/busybox requires the GNU style one. So let's sed adjust the format.